9.2/run-postgresql.sh

```diff
@@ -90,12 +107,16 @@ function initialize_database() {
 
 		# Allow connections from all hosts.
 		host all all all md5
+
+		# Allow replication connections from all hosts.
+		host replication all all md5
 	EOF
 
 	pg_ctl -w start
 	createuser "$POSTGRESQL_USER"
 	createdb --owner="$POSTGRESQL_USER" "$POSTGRESQL_DATABASE"
 	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH ENCRYPTED PASSWORD '${POSTGRESQL_PASSWORD}';"
+	psql --command "ALTER USER \"${POSTGRESQL_USER}\" WITH REPLICATION ;"
```

Comment:
Isn't this dangerous? According to http://www.postgresql.org/docs/9.2/static/sql-createrole.html:

> REPLICATION
> NOREPLICATION
> 
>    These clauses determine whether a role is allowed to initiate streaming
> replication or put the system in and out of backup mode. A role having the
> REPLICATION attribute is a very highly privileged role, and should only be
> used on roles actually used for replication. If not specified,
> NOREPLICATION is the default.

I don't have a lot of experience with replication, but maybe there should be a separate user for this, or just use the admin one? @praiskup any thoughts on this?